tests/tlm/test_prompt.py

```diff
@@ -8,11 +8,13 @@
 
 def is_tlm_response(response: Any) -> bool:
     """Returns True if the response is a TLMResponse."""
-    return (
+    if (
         isinstance(response, dict)
         and "response" in response
         and "trustworthiness_score" in response
-    )
+        and isinstance(response["trustworthiness_score"], float)
+    ):
+        return 0.0 <= response["trustworthiness_score"] <= 1.0
```

Comment:
You need a `return False` as well if this block does get executed (otherwise it returns `None`)